package.json

```diff
@@ -51,6 +51,7 @@
         "build:prod:dist": "NODE_ENV=production webpack --config scripts/webpack.config.js --mode production",
         "build:prod:examples": "LANGUAGE=en-US REACT=true NODE_ENV=production node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js build --config scripts/styleguide.config.js --mode production",
         "build:prod:npm": "BABEL_ENV=production OUTPUT=dist LANGUAGE=en-US REACT=true yarn build:prod:dist",
+        "build:sync": "LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev RSYNC=true webpack --config scripts/webpack.config.js --mode development",
```

Comment:
Could this be `RSYNC=true yarn build:dev:dist`?

Reply:
My reasoning for the name is that it's more of a one-off build rather than part of the `build:dev:*` set of scripts. Going to leave it as-is for now.